report/report.go

```diff
@@ -177,6 +189,14 @@ func MakeReport() Report {
 			WithShape(Pentagon).
 			WithLabel("daemonset", "daemonsets"),
 
+		StatefulSet: MakeTopology().
+			WithShape(Triangle).
+			WithLabel("stateful set", "stateful sets"),
+
+		CronJob: MakeTopology().
+			WithShape(Triangle).
+			WithLabel("cron job", "cron jobs"),
+
```

Comment:
Have we run out of shapes (again)? We really don't want these to be the same shape.